packages/graphql/src/graphql/ops/FElementData.graphql

```diff
@@ -1,14 +1,13 @@
 fragment ElementData on ElementInstance {
   elementData {
-    id
-    elementId
-    name
-    type
-    content
-    explanation
-    pointsMultiplier
-
     ... on ChoicesElementData {
```

Comment:
@rschlaefli  Do you know if there is a way to extract these shared attributes somewhere to only define them once? Unfortunately, the change to use union types for proper frontend types does not seem to support shared fields on these union types in the Pothos Schema and leaving them at the top level (as before) results in build errors for the graphql package